website/content/docs/connect/cluster-peering/usage/peering-traffic-management.mdx

```diff
@@ -0,0 +1,166 @@
+---
+layout: docs
+page_title: Cluster Peering L7 Traffic Management
+description: >-
+  Combine service resolver configurations with splitter and router configurations to manage L7 traffic in Consul deployments with cluster peering connections. Learn how to define dynamic traffic rules to target peers for redirects and failover. 
+---
+
+# L7 traffic management for cluster peering
```

Comment:
This is different from the sidebar "Manage L7 traffic with cluster peering"... which i feel is more clear. is there a reason why you chose this title?

i feel like the sidebar and the title should be the same (wherever possible). if the page title is too long, we can condense it in the sidebar title